### ISSUE TITLE
What is the point of offsetParent/offsetTop/offsetLeft?

### ISSUE THREAD

**Issue by domenic** [a11y-api/a11y-api#29]
These properties seem extraordinarily out of place, and encroaching on the territory of CSS. What purpose do they serve? The spec gives no normative requirements for how UAs are supposed to interpret them.

**Comment by domenic** [comment]
Talked about this F2F. It turns out there is a concept, maybe call it "accessibility x/y", which is used by screen readers to inform a few important features:

- Drawing a bounding box (useful for partially-sighted people)
- When sending a "virtual" click or similar, what (x, y) coordinates to use
- Scrolling the page to the appropriate (x, y)

So I think the spec needs to define the concept of "accessibility x/y" and state how offsetParent/top/left impact them. And also how accessibility x/y by default maps to some CSS concept of x/y.

**Comment by cookiecrook** [comment]
The whole API is accessibility specific, so it'd be redundant to use define "accessibility x/y" as an concept. We used the common terms "offset" terms familiar to web developers so that defining these would be intuitive.

**Comment by domenic** [comment]
You need to define the specific concept of X,y coordinates that are specific to accessibility technology. Using a generic name like "offset" is not acceptable there, so I suggested the prefix "accessibility". Remember these definitions live in the global namespace of all specs.

This is similar to how the spec defines accessible nodes, instead of just nodes.

The API, of course, doesn't need a prefix. Just the concept.

**Comment by cookiecrook** [comment]
Okay. Accessibility position and bounds are how these are referenced in Apple documentation. We'll need to reference the offset somehow because the concept of an element's positioning context is more unique to CSS or Web.

**Comment by minorninth** [comment]
Agreed that this needs a lot of explanation. Working on a whole section for this in the explainer.

Another big concern is it being async.
